server/storage/mvcc/index.go

```diff
@@ -107,6 +105,9 @@ func (ti *treeIndex) unsafeVisit(key, end []byte, f func(ki *keyIndex) bool) {
 	})
 }
 
+// Revisions returns limited number of revisions from key(included) to end(excluded)
+// at the given rev. The returned slice is sorted in the order of key. There is no limit if limit <= 0.
+// The second return parameter total isn't capped by the limit, and reflects all the expected revisions.
```

Comment:
Minor comment: replace total with `total`.

Reply:
@wayblink  Please update per the comment. Either replace total with `total`, or just remove the parameter name per @spzala 's comment. 

I will merge this PR once it's updated and pipeline is green.

Reply:
Updated. Thanks a lot